hermes-console-vue/src/api/inconsistent-group.ts

```diff
@@ -17,5 +17,5 @@ export interface InconsistentSubscription {
 
 export interface InconsistentMedata {
   datacenter: string;
-  content: string | undefined;
+  content?: string;
```

Comment:
When we have `content: string | undefined`, we can have `undefined` value, but the key `content` has to be present in the object, so in stubs we have to write: `content: undefined` (otherwise we have a type error).

In case of `content?: string`, we can both have object with key `content` with value `undefined` provided explicitly, but we also can omit this key completely (e.g. in test data, the value of absent key will still be `undefined` in JS).